recipes/aws-kvs-pic/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "cci.20210812":
+    url: "https://github.com/awslabs/amazon-kinesis-video-streams-pic/archive/bbf0e4ba749a6c045fc3958951c1c3a61a808dd0.zip"
```

Comment:
We prefer the `tar.gz` since it's smaller

Reply:
Done